server/src/main/java/org/apache/iotdb/db/qp/physical/crud/InsertRowPlan.java

```diff
@@ -196,13 +196,21 @@ public void markFailedMeasurementInsertion(int index, Exception e) {
   @Override
   public List<Path> getPaths() {
     List<Path> ret = new ArrayList<>();
-
     for (String m : measurements) {
       ret.add(new Path(deviceId, m));
     }
     return ret;
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    List<String> ret = new ArrayList<>();
+    for (String m : measurements) {
+      ret.add(new Path(deviceId, m).getFullPath());
```

Comment:
Just use `ret.add(deviceId + IoTDBConstant.PATH_SEPARATOR + m)`.

Reply:
I'll change it. Thanks for your suggestion.